### ISSUE TITLE
SET, DROP followed by any other statement fails with JS error

### ISSUE THREAD

**Issue by Phlosioneer** [mtxr/vscode-sqltools#651]
**Describe the bug**

Using the SET or DROP sql statement on its own works. However, if it is part of a series of statements, it fails with the error `Cannot destructure property 'name' of 'undefined' as it is undefined.`

**To Reproduce**
Connect to a database, and type something like:
```sql
USE innodb;
SET FOREIGN_KEY_CHECKS = 0;
```
Select both lines and run the query.

**Expected behavior**

Expected the query to be run with 0 returned rows.

**Screenshots**

**Desktop (please complete the following information):**
 - SQLTools Version 0.20.9
 - VSCode Version: 1.47.0
 - OS: Windows_NT x64 10.0.19041
 - Driver: 
   - [ ] PostgreSQL/Redshift
   - [x] MySQL/MariaDB
   - [ ] MSSQL/Azure
   - [ ] SQLite
   - [ ] Other? Which...
 - Database version: MySQL v5.7

I would be using 0.20.10 but I'm having the blank screen issue #648

**Additional context**

The set line by itself (`SET FOREIGN_KEY_CHECKS = 0;`) runs without error. Any query with multiple statements fails if _any_ of them are SET, regardless of where SET appears (top, middle, or bottom of list).

The set syntax must be valid for the issue to present. `SET foobar_baz = 0;` fails with ER_UNKNOWN_SYSTEM_VARIABLE, as expected. This makes me think the JS error is happening while parsing the response, rather than sending the request. 

I've only tested `DROP TABLE IF EXISTS <table name>` for the DROP error. I don't want to experiment with dropping tables. The drop line by itself runs without error, just like SET.

**Comment by JulioCesarAM** [comment]
i am having the same problem, did you manage to fix it ?

**Comment by bombillazo** [comment]
Not sure if related but I am trying to run this simple query:

![image](https://user-images.githubusercontent.com/44002321/96044822-07b59000-0e3f-11eb-85d8-d223a98d11ab.png)

But get an error: 
Cannot destructure property `name` of 'undefined' or 'null'.

**Comment by NCenerar** [comment]
Hello.

Probably related: When running single statements, it works:
```
DROP DATABASE IF EXISTS myproject;
```
```
CREATE DATABASE myproject;
```

but not when run together:
```
DROP DATABASE IF EXISTS myproject;
CREATE DATABASE myproject;
```

Fun fact: if I comment one line, it sill doesn't work which I find very weird...
Not working:
```
DROP DATABASE IF EXISTS myproject;
-- CREATE DATABASE myproject;
```
```
-- DROP DATABASE IF EXISTS myproject;
CREATE DATABASE myproject;
```


Thanks.

**Comment by gjsjohnmurray** [comment]
Will be fixed once #950 merges.
